__tests__/app.test.js

```diff
@@ -165,6 +163,23 @@ describe("GET/api/articles/:article_id", () => {
     });
 })
 
+describe("GET /api/users", () => {
+  test("200: Should respond with all users in an array of objects containing the properties username, name and avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        const {users} = body;
+        expect(users.length).toBeGreaterThan(0)
```

Comment:
As we have access to the test data, why can't we assert exactly how many there should be?